google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineStopAction.java

```diff
@@ -65,11 +65,14 @@ public void run() {
     GeneralCommandLine commandLine = new GeneralCommandLine(
         appEngineHelper.getGcloudCommandPath().getAbsolutePath());
 
-    commandLine.addParameters("preview", "app", "modules", "stop");
+    commandLine.addParameters("preview", "app", "versions", "stop");
+    commandLine.addParameters(versionToStop);
+
+    commandLine.addParameter("--service");
     for(String module : modulesToStop) {
       commandLine.addParameter(module);
     }
```

Comment:
Is it possible to give multiple modules (services) after `--service`? It doesn't seem so by the help message from `gcloud preview app versions stop --help`. (However, multiple versions are okay.)